tests/checks/mock/test_splunk_topology.py

```diff
@@ -0,0 +1,362 @@
+# stdlib
```

Comment:
Added a test for a FATAL message only. I'm kinda in the dark on the possible return types and combinations here.